metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/converter/ReferenceSequenceConverter.java

```diff
@@ -27,20 +28,33 @@
 
     @Override
     public ReferenceSequence convert(AssemblyType assemblyType) {
-        String refName = assemblyType.getNAME();
-        String patch = null;
-        // Attempt to extract patch from refName (only for GRC human or mouse assembly names)
-        if (refName != null && refName.matches("^GRC[hm]\\d+\\.p\\d+$")) {
-            String[] refNameSplit = assemblyType.getNAME().split("\\.", 2);
-            refName = refNameSplit[0];
-            patch = refNameSplit[1];
-        }
-        return new ReferenceSequence(
-                refName,
+        StringBuilder refName = new StringBuilder();
```

Comment:
You can initialize this like
```suggestion
        StringBuilder refName = new StringBuilder(assemblyType.getNAME());
```